commands/flame-save.js

```diff
@@ -0,0 +1,56 @@
+const { admin } = require('../firestore');
+
+module.exports = {
+  name: "flame-save",
+  description: "Saves flame to firestore",
+  execute(message, args) {
+
+    let user_id = ""; //outside bc docRef is const
+
+    try {
+      user_id = message.mentions.users.first().id;
```

Comment:
If no user is provided, will this still work?

Reply:
yes